overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -2597,7 +2605,7 @@ func (s *deviceMgrRemodelSuite) TestRemodelUC20SwitchKernelBaseGadgetSnapsInstal
 		"label":     expectedLabel,
 		"directory": filepath.Join(boot.InitramfsUbuntuSeedDir, "systems", expectedLabel),
 		// none of the tasks are downloads so they were not tracked
-		"snap-setup-tasks": nil,
+		"snap-setup-tasks": []interface{}{"1", "4", "6"},
```

Comment:
Nitpick, maybe use `aTask.ID(), bTask.ID(), ...` if the test has vars for them already, or add a comment explaining what tasks they are?